.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: node_js
-
-node_js:
-- '14'
-
-install:
-- TMPDIR=/tmp yarn install
-
-script:
-- yarn $TEST_SUITE
-
-cache: false
-
-addons:
-  chrome: stable
-
-env:
-  matrix:
-  - TEST_SUITE=lint:ci
-  - TEST_SUITE="test -t karma"
-  - TEST_SUITE=test:tsd:ci
-  - TEST_SUITE=test:testcafe-ci
-  - TEST_SUITE="test -t jest"
```

Comment:
Is removing this file OK?

Reply:
It's to avoid unnecessary wait after bacon tasks are green, removal of this file should only happen in 0.0 topic branch, it will be added back when merge back to master

Reply:
Disabled the suites and removed the "chrome" addon instead of deleting file. Travis hangs if there is no config present